src/lib/markbind/src/parser.js

```diff
@@ -804,34 +604,32 @@ class Parser {
     // rebase current element
     if (element.attribs[ATTRIB_INCLUDE_PATH]) {
       const filePath = element.attribs[ATTRIB_INCLUDE_PATH];
-      let newBase = Parser.calculateNewBaseUrls(filePath, this.rootPath, this.baseUrlMap);
-      if (newBase) {
-        const { relative, parent } = newBase;
+      let newBaseUrl = urlUtils.calculateNewBaseUrls(filePath, this.rootPath, this.baseUrlMap);
```

Comment:
The only irrelevant refactor to remove the nested `ifs`, and slightly better variable naming; Let me know if this should be removed!